dev-docs/bidders/aniview.md

```diff
@@ -5,13 +5,17 @@ description: Prebid Aniview Bidder Adapter
 pbjs: true
 biddercode: aniview
 media_types: banner, video
+gpp_supported: true
```

Comment:
Sorry, this field has been deprecated. As described in https://docs.prebid.org/dev-docs/bidder-adaptor.html, this is now a list of which GPP section IDs are supported. e.g. gpp_sids: tcfeu, tcfca, usnat, usstate_all